docs/setting-snap-channel.md

```diff
@@ -0,0 +1,193 @@
+```
+
+In this case you will only receive updates for the 1.16 release of Kubernetes,
+and MicroK8s will never upgrade to 1.17, unless you explicitly
+[refresh the snap](#refresh)
+
+
+## Stable, candidate, beta and edge releases
+
+The `*/stable` channels serve the latest stable upstream Kubernetes release of
+the respective release series. Upstream releases are propagated to the MicroK8s
+snap in about a week. This means your MicroK8s will upgrade to the latest
+upstream release in your selected channel roughly one week after the upstream
+release.
+
+The `*/candidate` and `*/beta` channels get updated within hours of an upstream
+release. Getting a MicroK8s deployment pointing to `1.16/beta` is as simple as:
+
+```
+snap install microk8s --classic --channel=1.16/beta
+```
+
+The `*/edge` channels get updated on each MicroK8s patch or upstream
+Kubernetes patch release.
+
+Keep in mind that edge and beta are snap constructs and do not relate to
+specific Kubernetes release names.
+
+
+## Tracks with pre-stable releases
+
+On tracks where no stable Kubernetes release is available, MicroK8s ships
+pre-release versions under the following scheme:
+
+-   The `*/edge` channel (eg `1.16/edge`) holds the alpha upstream releases. 
+-   The `*/beta` channel (eg `1.16/beta`) holds the beta upstream releases.
+-   The `*/candidate` channel (eg `1.16/candidate`) holds the release candidate
+    of upstream releases.
+
+Pre-release versions will be available the same day they are released upstream. 
+
+For example, to test your work against the alpha `v1.16` release simply run:
+
+```
+sudo snap install microk8s --classic --channel=1.16/edge
+```
+
+However, be aware that pre-release versions may require you to configure the
+Kubernetes services on your own.
+
+
+## I am confused. Which channel is right for me?
+
+The single question you need to focus on is what channel should be used below:
+
+```
+sudo snap install microk8s --classic --channel=<which_channel?>
+```
+
+Here are some suggestions for the channel to use based on your needs:
+
+-   I want to always be on the latest stable Kubernetes.
+
+     -- Use `--channel=latest`
+
+-   I want to always be on the latest release in a specific upstream K8s release.
+
+     -- Use `--channel=<release>/stable`, eg `--channel=1.14/stable`. 
+
+-   I want to test-drive a pre-stable release.
+
+     -- Use `--channel=<next_release>/edge` for alpha releases.
+
+     -- Use `--channel=<next_release>/beta` for beta releases.
+
+     -- Use `--channel=<next_release>/candidate` for candidate releases.
+
+-   I am waiting for a bug fix on MicroK8s:
+
+     -- Use `--channel=<release>/edge`.
+
+-   I am waiting for a bug fix on upstream Kubernetes:
+
+     -- Use `--channel=<release>/candidate`.
+
+
+<a id="refresh"> </a>
+## Changing channels
+
+If you have installed MicroK8s already, and then decide you wish to change
+to a particular channel, you can use the `snap refresh` command to switch to a
```

Comment:
Same here. Lets ask them to remove and reinstall. Or tell them that refresh will potentially break them. @tvansteenburgh this is something we could work on.

Reply:
I have added a big warning. I think people will look for how to change channels, so to omit it will not be helpful. And there are some valid use cases which should be safe(?) i.e. moving between stable and candidate or whatever. Anyhow, I have added a large warning, let me know what you think of the wording